src/modules/flow/http-server/http-server.json

```diff
@@ -498,6 +504,143 @@
       ],
       "private_data_type": "http_data",
       "url": "http://solettaproject.org/doc/latest/node_types/http-server/direction-vector.html"
+    },
+    {
+      "category": "output/network",
+      "description": "HTTP Server for Blob.",
```

Comment:
On 29-03-2016 11:15, Gustavo Lima Chaves wrote:
> In src/modules/flow/http-server/http-server.json
> <https://github.com/solettaproject/soletta/pull/1752#discussion_r57729943>:
>
>> @@ -498,6 +504,143 @@
>>        ],
>>        "private_data_type": "http_data",
>>        "url": "http://solettaproject.org/doc/latest/node_types/http-server/direction-vector.html"
>> +    },
>> +    {
>> +      "category": "output/network",
>> +      "description": "HTTP Server for Blob.",
>
> We got to explain (much) better what our nodes do here :/

Http Server for Bob. If you're not Bob, move over =P

>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/solettaproject/soletta/pull/1752/files/9cdba8487a04389cc679e2245df64fb7449fa680#r57729943>
>